src/pages/Browse/index.jsx

```diff
@@ -325,19 +333,40 @@ const Browse = () => {
                       <td>{data?.uploaddate.split(".")[0]}</td>
                     </tr>
                   ))}
-                <tr>
-                  <td colSpan="6">
-                    Page:
-                    {pagesOptions && (
-                      <InputContainer
-                        name="page"
-                        type="select"
-                        onChange={(e) => handleChange(e)}
-                        options={pagesOptions}
-                        property="value"
-                        className="mt-n4"
-                      />
-                    )}
+                <tr className="text-left">
+                  <td />
+                  <td />
+                  <td />
+                  <td />
+                  <td />
+
+                  <td>
```

Comment:
Would recommend using `<td colspan="6">` with content alignment to right for better UX.

Reply:
Used this method +1